extensions/azurecore/src/azureResource/providers/databaseServer/databaseServerTreeDataProvider.ts

```diff
@@ -72,7 +72,8 @@ export class AzureResourceDatabaseServerTreeDataProvider implements azureResourc
 					saveProfile: false,
 					options: {}
 				},
-				childProvider: 'MSSQL'
+				childProvider: 'MSSQL',
+				type: 'Server'
```

Comment:
Type should be an enum

Reply:
We went over this in our last discussion. We want it to be a string to be more generalized.

Reply:
That's fine to have the actual property be a string but I would still rather see a string enum made for the types you're using here so we aren't hardcoding the names in multiple places.

Reply:
yeah, I agree with @Charles-Gagnon suggestion.  Unless there is a need to allow arbitrary types I would recommend using an enum of strings